libraries/AP_HAL_ChibiOS/hwdef/Pixhawk4/hwdef.dat

```diff
@@ -3,6 +3,14 @@
 
 include ../fmuv5/hwdef.dat
 
+# red and blue LEDs swapped vs fmuv5
+# red LED marked as B/E
+undef PC7
+PC7 LED_RED OUTPUT GPIO(92) HIGH
```

Comment:
This should be GPIO(90), keeping it 92 still ties PC7 to GPIO_B_LED , Pin name is not relevant.

Reply:
great, thanks for the advice! fixed!